src/Compilers/CSharp/Portable/Symbols/Source/SourceTypeParameterSymbol.cs

```diff
@@ -171,12 +177,15 @@ internal override CustomAttributesBag<CSharpAttributeData> GetAttributesBag()
                 var sourceMethod = this.ContainingSymbol as SourceMemberMethodSymbol;
                 if ((object)sourceMethod == null || (object)sourceMethod.SourcePartialDefinition == null)
                 {
-                    lazyAttributesStored = LoadAndValidateAttributes(OneOrMany.Create(this.MergedAttributeDeclarationSyntaxLists), ref _lazyCustomAttributesBag);
+                    lazyAttributesStored = LoadAndValidateAttributes(
```

Comment:
@AlekseyTs OK, that makes more sense. I do think that this is a problem but is out of the scope of this PR.

I'm going to file another bug for fixing this since it will require some significant refactoring of parameter symbols.